.github/workflows/tests.yml

```diff
@@ -0,0 +1,76 @@
+name: "build"
+
+on:
```

Comment:
I think we should follow https://github.com/hypertrace/goagent/blob/main/.github/workflows/pr-test.yml#L2